clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -917,6 +917,26 @@ default ListConsumerGroupsResult listConsumerGroups() {
         return listConsumerGroups(new ListConsumerGroupsOptions());
     }
 
+    /**
```

Comment:
I would put all of the share groups methods (in all of these classes) below the consumer group method block. You've put them in the middle of the consumer group methods so far.